### ISSUE TITLE
[Translation] 1_concurrent_vs_parallel.md

### ISSUE THREAD

**Issue by taejs** [cfsamson/book-exploring-async-basics#36]
> There are several ways to multitask. One is by progressing tasks concurrently, but not at the same time. Another is to progress two tasks at the exact same time in parallel.

In this sentence, I think it's better to change the word `two tasks` to `multiple tasks`.
Because progressing multiple tasks is also called in parallel, not only progressing two tasks.

**Comment by taejs** [comment]
I have another question about that.
Are there only ways to multitask are just two of them? (progressing tasks concurrently and parallel)
Then, how about change the word `several` to `two`?

**Comment by cfsamson** [comment]
>In this sentence, I think it's better to change the word two tasks to multiple tasks.

I agree, it's better to skip "two" in the second sentence so it doesn't give the impression that parallel is only applicable if there are two tasks. I'll change that.

>Are there only ways to multitask are just two of them? (progressing tasks in parallel and concurrently)
Then, how about change the word several to two?

Good question. You're probably right that I could just write "There are two ways to multitask".

**Comment by taejs** [comment]
> Once we create more threads than there are cores, the OS will switch between our threads and progress

I need more explanation about it to make clear for me. 
What does the word `progress` mean? 
It wasn't defined anywhere.

**Comment by cfsamson** [comment]
>I need more explanation about it to make clear for me.
What does the word progress mean?
It wasn't defined anywhere.

If you take a look at the definition of the word [progress](https://dictionary.cambridge.org/dictionary/english/progress) it means to move forward towards a goal. When I say "progress a thread" I mean to advance it's state. In other words, make progress on the task executing in the thread. 

When an OS switches between threads using which is usually what happens in all operating systems employing [preemtive multitasking](https://cfsamson.github.io/book-exploring-async-basics/2_async_history.html#preemptive-multitasking) (which you'll read about in chapter 2) it will take turns on giving the different threads real CPU time.

**Comment by taejs** [comment]
It's almost done! Thanks for the great works. It really helps with understanding concurrency.

Would you confirm that I understand it correctly?
It's about the word `reference frame`. I think that means the context that we have to share before we start. The reason I am asking is that there's no word perfectly fitted in Korean. So I have to find an alternative word.

**Comment by cfsamson** [comment]
No problem :)

>It's about the word reference frame. I think that means the context that we have to share before we start. The reason I am asking is that there's no word perfectly fitted in Korean. So I have to find an alternative word.

Yeah, I guess that is a pretty good understanding. If you're looking at translations the term "reference frame" is often used in physics to describe how systems work in relation to each other, like in the theory of Special Relativity. You could actually see if the article about [Special Relativity](https://en.wikipedia.org/wiki/Special_relativity) is translated to Korean and you should find a good translation of the term there.
